addons/uh60_mfd/functions/fnc_restorePylons.sqf

```diff
@@ -0,0 +1,17 @@
+/*
+ * vtx_uh60_mfd_fnc_restorePylons
+ *
+ * Restores pylon values used in MFDs for recall during reload events
+ *
+ * params (array)[(object) vehicle]
+ */
+
+params ["_vehicle"];
+
+{
+	private _index = _x;
+
+	_vehicle setPylonLoadout [_index select 0, "vtx_1000rnd_dummy", true];
+	_vehicle setAmmoOnPylon [_index select 0, _index select 1];
```

Comment:
`[_index select 0, _index select 1]` this is just `_x` right?

Reply:
Correct. I got in the habit of putting _x into it's own variable due to nested forEach loops, but it's not strictly necessary here. I'll just use _x.